lib/sisimai/bite/email/ezweb.rb

```diff
@@ -0,0 +1,260 @@
+module Sisimai::Bite::Email
+  module EZweb
+    # Sisimai::Bite::Email::EZweb parses a bounce email which created by au EZweb.
+    # Methods in the module are called from only Sisimai::Message.
+    class << self
+      # Imported from p5-Sisimail/lib/Sisimai/Bite/Email/EZweb.pm
+      require 'sisimai/bite/email'
+
+      Re0 = {
```

Comment:
__[rubocop]__
* Style/MutableConstant: Freeze mutable objects assigned to constants.

<!-- Comment by [SideCI](https://sideci.com) -->